packages/cozy-doctypes/src/contacts/Contact.js

```diff
@@ -116,6 +125,44 @@ class Contact extends Document {
   static getDisplayName(contact) {
     return Contact.getFullname(contact) || Contact.getPrimaryEmail(contact)
   }
+
+  /**
+   * Returns personal data for the contact
+   *
+   * @param {Object} contact - A contact
+   * @param {Array} fields - The list of fields to retrieve
+   * @return {Object} - the personal data
+   **/
+  static getPersonalData(contact, fields) {
```

Comment:
I thought this would have been in a Procedure model rather than contact, since the personal data names are really coupled to procedures?

Reply:
Good point! And I think we actually said that when we talked about it a few days ago. @Crash-- do you agree?

Reply:
ok 👍